packages/shared/src/bounties.ts

```diff
@@ -1302,9 +1317,23 @@ export async function bountyClosureActiveTest<
     .toMatchSnapshot('bounty canceled event')
 
   // verify the curator transfer of balance event to the treasury
-  await checkSystemEvents(client, { section: 'balances', method: 'Transfer' })
```

Comment:
@dhirajs0 this was capturing some fee events, which caused spurious test failures. Might help to know this for treasury tests!